compiler/plugins/target/ROCM/builtins/ukernel/iree_uk_amdgpu_multi_mma_mfma_i32_16x16x32_i8.c

```diff
@@ -6,46 +6,140 @@
 
 #include "compiler/plugins/target/ROCM/builtins/ukernel/common.h"
 
-// Very naive kernel. TODO(bjacob):
-// 1. Shared memory: can't allocate it within the microkernel (which is just a
-//    helper device function, not the actual amdgpu_kernel). Need to get it
-//    passed down here as additional parameters.
-// 2. Better scheduling via either barrier intrinsics or inline assemby.
+// Microkernel for iree_gpu.multi_mma with DataTiledMMAAttr with
+// intrinsic = MFMA_I32_16x16x32_I8 and a shape with outer M and N dimensions
+// equal to 1 (so that this is just doing the inner loop on the K dimension).
+//
+// This microkernel uses a local memory workspace buffer provided by the caller.
```

Comment:
Without the context, I wouldn't be able to tell if you refer to shared or private memory here. I'd either call it *LDS* workspace buffer, sticking to the AMD naming, or *shared memory*/*workgroup memory* workspace, sticking to more popular terms in the industry: https://github.com/nod-ai/shark-ai/blob/main/docs/amdgpu_kernel_optimization_guide.md#glossary

Reply:
I think most of the other passes in IREE call it shared memory